### DISCUSSION TITLE
版主置顶吧，共享要尽头了，以后都是API的天下

### DISCUSSION THREAD

**Discussion by ixxmu** [chatgpt-web-dev/chatgpt-web#433] [General]
版主置顶吧，共享access token 要尽头了，不是一个ninja能搞定的，都别折腾了！

不信邪的继续吧

**Comment by BobDu** [comment]
确实越来越有这个趋势了。逆向难度一直在上升。API又再降价。
如果API价格能再降几轮以后。 共享access token 可能就越来越没有必要了。
